poc_iot_verifier/src/runner.rs

```diff
@@ -423,3 +436,23 @@ impl Runner {
         Ok(())
     }
 }
+
+fn poc_challengee_reward_unit(num_witnesses: u32) -> Result<Decimal> {
+    if num_witnesses == 0 {
+        Ok(Decimal::ZERO)
+    } else if num_witnesses < WITNESS_REDUNDANCY {
+        Ok(Decimal::from(WITNESS_REDUNDANCY / num_witnesses))
+    } else {
+        let exp = WITNESS_REDUNDANCY - num_witnesses;
+        if let Some(to_sub) = POC_REWARD_DECAY_RATE.checked_powu(exp as u64) {
+            let unnormalized = Decimal::TWO - to_sub;
+            Ok(normalize_reward_unit(unnormalized))
+        } else {
+            Err(Error::InvalidExponent(exp.to_string()))
+        }
+    }
+}
+
+fn normalize_reward_unit(unnormalized: Decimal) -> Decimal {
+    std::cmp::min(HIP15_TX_REWARD_UNIT_CAP, unnormalized)
```

Comment:
nit: Just move this to the above function directly, there's no need to have `normalized_reward_unit` function separately if we're just calling it from a single place.

Reply:
fixed [d613888](https://github.com/helium/oracles/pull/205/commits/d61388894263863c2d4eb13c992e96ca7330698f)